### PR TITLE
irc: Dial down the SPAM filter a bit

### PR DESCRIPTION
**Pull request by KellerFuchs**

### CODE REVIEW

File: vault/irc/vault.yml [hunk 1]
```diff
@@ -1,140 +1,143 @@
 $ANSIBLE_VAULT;1.1;AES256
-62636134366666616663626662656138666633333738376364306439333333343638623139613930
```
Comment (mayli) [inline vault/irc/vault.yml:2]: lgtm, yes i can read those numbers.

**Review by mayli — APPROVED** [review]

File: vault/irc/vault.yml (hunk 1 above)
Comment (KellerFuchs) [reply vault/irc/vault.yml:2]: Pfffff

**Review by DeviaVir — DISMISSED** [review]
Did someone really type a sentence that just so happened to have all of those letters in that sequence? That's amazing, but fair enough this looks more impossible to run into.

**Comment by KellerFuchs** [comment]
@DeviaVir Yeah, it will also fail to stop some of the SPAM which has happened.

**Comment by KellerFuchs** [comment]
@hashbang/administrators Could someone fix the escaping (read: add it) in [`spamfilter.conf.j2`](https://github.com/hashbang/admin-tools/blob/master/files/unrealircd/spamfilter.conf.j2#L204) ?

**Comment by KellerFuchs** [comment]
@hashbang/administrators Also, please review/merge.
The escaping fix can happen later.

**Review by DeviaVir — APPROVED** [review]

**Comment by KellerFuchs** [comment]
@DeviaVir Thanks for the review.

**Pull request merged**